low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -24,6 +24,9 @@
 import time
 import sys
 
+from cortx.utils.message_bus import MessageProducer
```

Comment:
Instead of changing this file, its better to create new with just required code under
framework/messagebus/mb_egress_processor.py

@ujjwalpl Here too message bus is replacing rabbitmq functionality totally(like we discussed in case of plane_controller_handler in afternoon today), so if we go by that policy then rabbitmq code should be removed totally? Or should retain to keep backward compatibility. In my opinion, latter suits better.